validate/validate.go

```diff
@@ -96,7 +96,11 @@ func NewValidatorFromPath(bundlePath string, hostSpecific bool, platform string)
 // CheckAll checks all parts of runtime bundle
 func (v *Validator) CheckAll() (msgs []string) {
 	msgs = append(msgs, v.CheckPlatform()...)
-	msgs = append(msgs, v.CheckRootfsPath()...)
+	if v.platform == "windows" && v.spec.Windows.HyperV != nil {
+		msgs = append(msgs, fmt.Sprintf("for Hyper-V containers, Root must not be set"))
```

Comment:
I think we're missing a check here, and should be using:

```go
if v.platform == "windows" && v.spec.Windows.HyperV != nil {
  if v.spec.Root != nil {
    msgs = append(msgs, fmt.Sprintf("for Hyper-V containers, Root must not be set"))
  }
} else { …
```

Otherwise we'll be erroring on all Hyper-V configs, not just those that set `root`.